README.md

```diff
@@ -143,14 +143,17 @@ fastify.ready(err => {
       path: './examples/example-static-specification.yaml',
       postProcessor: function(swaggerObject) {
         return swaggerObject
-      }
-    }
+      },
+      baseDir: '/path/to/external/spec/files/location',
+    },
   }
   ```
   Example of the `fastify-swagger` usage in the `static` mode is available [here](examples/static-file.js).
 
   `specification.postProcessor` parameter is optional. It allows you to change your swagger object on the fly (for example - based on the environment). It accepts `swaggerObject` - basically a javascript object which was parsed from your `yaml` or `json` file and should return a swagger object.
 
+  `specification.baseDir` allows to specify directory where all spec files which are included into the main one using `$ref` live. 
+  By default it's the directory where main spec file is located. Should be absolute path without trailing slash.
```

Comment:
> Provided value should be an absolute path without trailing slash.

Why don't we trim the trailing slash if exists, instead of relying on user to know this?

Reply:
@cemremengu yep, you are right, I forgot to implement it. Will fix in the next commit. And on the other hand keeping this as part of documentation (even with trimming trailing slashes) would make it more obvious for the users how to use it (IMO)

Reply:
Done